aquarius/events/events_monitor.py

```diff
@@ -91,9 +91,13 @@ def __init__(self, web3, config_file, metadata_contract=None):
         if self._ecies_private_key:
             self._ecies_account = Account.privateKeyToAccount(self._ecies_private_key)
 
+        ignore_last_block = get_bool_env_value("IGNORE_LAST_BLOCK", 0)
```

Comment:
can we have unit tests and documentation for this option?

Reply:
agreed.